src/components/Subheading/README.md

```diff
@@ -46,12 +46,27 @@ Subheadings should follow the content guidelines for [headings and subheadings](
 
 ## Examples
 
+### Typographic subheading
+
+Used for the title of any sub-sections in top-level page sections.
```

Comment:
Following the same sort of wording as `Heading`.

This also removes the mentions of Card so this example can feel isolated.